manoseimas/compatibility_test/client/app/ResultsView/SimilarityTopics/index.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { PropTypes } from 'react'
+import { connect } from 'react-redux'
+import { saveAnswer,
+         getResults,
+         toggleImportance,
+         showHeader,
+         hideHeader,
+         setTab,
+         expandFraction } from '../../../store/modules/results'
+import Topics from './Topics'
+import Loader from 'react-loader'
+import styles from '../../../styles/views/results.css'
+
+class SimilarityTopicsContainer extends React.Component {
+
+    static propTypes = {
+      user_answers: PropTypes.object,
+      fractions: PropTypes.array,
+      mps: PropTypes.array,
+      topics: PropTypes.array,
+      saveAnswer: PropTypes.func,
+      getResults: PropTypes.func,
+      toggleImportance: PropTypes.func,
+      show_header: PropTypes.bool,
+      showHeader: PropTypes.func,
+      setTab: PropTypes.func,
+      hideHeader: PropTypes.func,
```

Comment:
Why do you add this prop here if you don't use it?
Same question to a few other props here.